guides/common/modules/con_defining-content-access-strategies-for-hosts.adoc

```diff
@@ -0,0 +1,19 @@
+[id="defining-content-access-strategies-for-hosts-{context}"]
+= Defining content access strategies for hosts
+
+When defining your content lifecycle in {Project}, you can use content views and lifecycle environments to define which hosts can access which content and content versions.
+
+{Project} includes _Default Organization View_ and the _Library_ lifecycle environment by default.
+
+Default Organization View::
+The _Default Organization View_ is the default content view in {Project} that contains all the content that is synchronized to {Project}.
+After you update your content, such as by adding or removing a repository, the update is immediately reflected in _Default Organization View_.
+
+Library::
+The _Library_ lifecycle environment is the default lifecycle environment in {Project} that always contains the latest content from the repositories in a given content view.
```

Comment:
The user can promote and content view version to use the Library lifecycle environment. Maybe it would be better to say instead that every newly published content view version is automatically published to the Library lifecycle environment.